src/components/Tag/Tabs/VulnerabilitiesDetails.jsx

```diff
@@ -81,6 +106,9 @@ function VulnerabilitiesDetails(props) {
   const [isEndOfList, setIsEndOfList] = useState(false);
   const listBottom = useRef(null);
 
+  const [anchorEl, setAnchorEl] = useState(null);
```

Comment:
maybe use a more suggestive name for these variables that are related to the popper